template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/atlas/AtlasKnoxRoleConfigProvider.java

```diff
@@ -108,7 +110,8 @@ public boolean isDatalakeVersionSupported(String blueprintText) {
 
     private boolean isShapeVersionSupportedByRuntime(String runtime) {
         Comparator<Versioned> versionComparator = new VersionComparator();
-        return versionComparator.compare(() -> runtime, () -> MIN_VERSION_FOR_DIFFERENCIAL_BACKUP) > -1;
+        return versionComparator.compare(() -> runtime, () -> MIN_VERSION_FOR_DIFFERENCIAL_BACKUP) > -1
```

Comment:
Is there any way to have a unit test for this? Just to ensure it is not changed in the future?